src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -335,6 +337,9 @@ def makeTransferRequest(self, wflow):
                         self.dsetCounter += 1
                     else:
                         self.blockCounter += len(blocks)
+                else:
+                    self.logger.info("FAKE data subscription. Default transfer id set to: '1111'")
+                    transRec['transferIDs'].add("1111")  # any fake number
```

Comment:
I"m not sure why do you need fake numbers at all? If there is no transfer it should not be any IDs either. Better you should put `None` as transferIDs which resolve any ambiguity. Then, in other parts of the code, you should check that transferIDs should be not a `None` which is the only right criteria for IDs. All other IDs should be considered as possible values.

Reply:
I think I had a fake ID in order to test the whole cycle between Transferor and Monitor. At this stage they are indeed no longer needed, especially because we're about to set things in write mode.
That field is supposed to contain a list of transfer IDs, so if there is none, it's better to set it to an empty string.
Thanks for your comments though